src/app/pages/properties/my-wishlist/my-wishlist.component.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { async, ComponentFixture, TestBed, fakeAsync, tick } from '@angular/core/testing';
+
+import { MyWishlistComponent } from './my-wishlist.component';
+import { HttpClientModule } from '@angular/common/http';
+import { RouterTestingModule } from '@angular/router/testing';
+import { PropertiesWishlistService } from 'src/app/services/properties/properties-wishlist/properties-wishlist.service';
+import { propertiesWishlistSpy, toastServiceSpy, routerSpy } from 'src/app/helpers/spies';
+import { ToastrService } from 'ngx-toastr';
+import { Router } from '@angular/router';
+import { of, throwError } from 'rxjs';
+import { By } from '@angular/platform-browser';
+
+describe('MyWishlistComponent', () => {
+  let component: MyWishlistComponent;
+  let fixture: ComponentFixture<MyWishlistComponent>;
+  const mockWihslistProperties = {
```

Comment:
What do you think of having this response in the `src/app/shared/mocks.ts` file and then import it here

Reply:
Done